src/main/java/nextstep/subway/line/domain/LineColor.java

```diff
@@ -14,13 +15,21 @@ protected LineColor() {
     }
 
     private LineColor(String value) {
+        validate(value);
         this.value = value;
     }
 
     public static LineColor from(String value) {
         return new LineColor(value);
     }
 
+    private void validate(String value) {
+        if (StringUtils.hasText(value)) {
+            return;
+        }
+        throw new IllegalArgumentException("라인 색상은 빈 값일 수 없습니다.");
```

Comment:
```suggestion
        if (!StringUtils.hasText(value)) {
            throw new IllegalArgumentException("라인 색상은 빈 값일 수 없습니다.");
        }
```
보통은 이런식으로 구성하지 않을까요? return; 도 없앨수 있구요

Reply:
not을 안쓰기 위해 했는데, 함수 자체가 영향이 크게 없는거 같습니다.
참고해서 수정하겠습니다!